client/views/create/create.js

```diff
@@ -90,6 +90,9 @@ Template.create.events({
     const maxResponse = responseSelect[responseSelect.selectedIndex].value;
     const hiddenSelector = document.getElementsByName('visibility')[0];
     const isHidden = (hiddenSelector[hiddenSelector.selectedIndex].value === 'hidden');
+    const socialSelector = document.getElementsByName("social")[0]; // Extract the HTML select element for getting the social toggles
+    const isSocial = socialSelector[socialSelector.selectedIndex].value === "on"; // Boolean value to store whether the user has selected Social share to be on or off
```

Comment:
The same here, use `'on'`, just to be consistent with the rest of the code in the file.

Reply:
Updated the quotes here.